WordPress/Classes/Utility/ContextManager.h

```diff
@@ -16,8 +16,8 @@ FOUNDATION_EXTERN NSString * const ContextManagerModelNameCurrent;
 - (void)saveContextAndWait:(NSManagedObjectContext *)context;
 - (void)saveContext:(NSManagedObjectContext *)context;
 - (void)saveContext:(NSManagedObjectContext *)context withCompletionBlock:(void (^)(void))completionBlock;
-- (void)saveUsingBlock:(void (^)(NSManagedObjectContext *context))aBlock;
-- (void)saveUsingBlock:(void (^)(NSManagedObjectContext *context))aBlock completion:(void (^)(void))completion;
+- (void)performAndSaveUsingBlock:(void (^)(NSManagedObjectContext *context))aBlock;
+- (void)performAndSaveUsingBlock:(void (^)(NSManagedObjectContext *context))aBlock completion:(void (^)(void))completion;
```

Comment:
> Same – I've heard it's a hard thing 😁. If you're good with `performAndSave`, I am too – great artists steal, and all that!

LGTM. 

Well... I thought of `performThenSave...` just to make it clear which happens first, but maybe I'm splitting hairs here 😅 .